python/cudf/cudf/core/dataframe.py

```diff
@@ -1456,6 +1456,98 @@ def add(self, other, axis="columns", level=None, fill_value=None):
 
         return self._apply_op("add", other, fill_value)
 
+    def update(
+        self,
+        other,
+        join="left",
+        overwrite=True,
+        filter_func=None,
+        errors="ignore",
+    ):
+        """
+        Modify a DataFrame in place using non-NA values from another DataFrame.
+
+        Aligns on indices. There is no return value.
+
+        Parameters
+        ----------
+        other : DataFrame, or object coercible into a DataFrame
+            Should have at least one matching index/column label with the
+            original DataFrame. If a Series is passed, its name attribute must
+            be set, and that will be used as the column name to align with the
+            original DataFrame.
+
+        join : {‘left’}, default ‘left’
```

Comment:
I'm always bad at sphinx marks, should we use the quotation mark or backtick here?